src/main/java/com/malinatran/ClientHandler.java

```diff
@@ -1,16 +1,24 @@
 package com.malinatran;
 
-import java.lang.Runnable;
+import com.malinatran.reader.Reader;
+import com.malinatran.writer.Writer;
+import com.malinatran.request.Request;
+import com.malinatran.request.RequestListener;
+import com.malinatran.response.Response;
+import com.malinatran.router.Router;
+
 import java.io.*;
```

Comment:
We should be explicit about what we'd like to import from this package since we don't need everything from java.io